schema/json/schema.json

```diff
@@ -40,6 +40,9 @@
                             "architecture": {
                                 "type": "string"
                             },
+                            "author": {
```

Comment:
this means it accepts both "author" and "authors" ?

Reply:
yeah, this section is shared with all metadata's... one has author, another has authors. (this looks valid to me)